src/main/java/com/icthh/xm/tmf/ms/activation/service/ResponseEnricherServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.icthh.xm.tmf.ms.activation.service;
+
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
+import com.icthh.xm.tmf.ms.activation.domain.SagaTransaction;
+import com.icthh.xm.tmf.ms.activation.mapper.ServiceMapper;
+import com.icthh.xm.tmf.ms.activation.model.v4.Service;
+import com.icthh.xm.tmf.ms.activation.model.v4.ServiceCreate;
+import com.icthh.xm.tmf.ms.activation.resolver.TransactionTypeKeyResolver;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@LepService(group = "service.resource")
+@org.springframework.stereotype.Service
```

Comment:
@RomanTeren 
`@LepService` already contains spring `@Service`. don not need to add again

Reply:
https://github.com/xm-online/tmf-ms-activation/pull/94